workflow/envs/savana.yaml

```
@@ -0,0 +1,13 @@
+  - conda-forge
+  - bioconda
+  - nodefaults
+dependencies:
+  - pybedtools==0.9.0
+  - pysam==0.20.0
+  - cyvcf2==0.30.16
+  - scikit-learn==1.2.2
+  - pandas==2.0.0
+  - matplotlib==3.7.1
+  - pip:
+    - git+https://github.com/johanneskoester/savana.git@fix/empty-calls
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)